target/linux/realtek/files-5.15/arch/arm64/boot/dts/realtek/rtd1295-giraffe-emmc-2gb.dts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017-2019 Andreas Färber
```

Comment:
No, this is for our standard evaluation board. Andrea only commit some COST boards manufactured by our customers, such as: rtd1295-mele-v9.dts, rtd1295-probox2-ava.dts and rtd1295-zidoo-x9s.dts. These files has been upstreamed since kernel 5.8. I just copy the template from these files, so that I keep the author name in this file. The similar condition exists for rtd1296 as well.

Reply:
Yeah, not sure if copyright works that way, if you wrote it from scratch, its your copyright. If you copied it from andreas, I think you still need your copyright, and then add something like 'based on XXX'. But this is all legalese :p